subsys/net/openthread/Kconfig

```diff
@@ -35,6 +35,7 @@ config OPENTHREAD_NRF_SECURITY_PSA
 	select NRF_SECURITY
 	select MBEDTLS_ENABLE_HEAP if !OPENTHREAD_COPROCESSOR_RCP
 	select MBEDTLS_TLS_LIBRARY if (OPENTHREAD_JOINER || OPENTHREAD_COMMISSIONER || OPENTHREAD_COAPS)
+	select MBEDTLS_PEM_PARSE_C if (OPENTHREAD_COAPS || OPENTHREAD_BLE_TCAT)
```

Comment:
@frkv I notice the other selects below are using a `select PSA_WANT_ALG_*` scheme.
iirc in order to both avoid the stuck-symbol syndrome as well as circular deps in Kconfig security settings.

Should the `MBEDTLS_PEM_PARSE_C` have a `WANT` equivalent ?

Reply:
I agree it would be good to create an `PSA_WANT` equivalent, but I think it can be done separately by Crypto team.